ledger/ledger_app.md

```diff
@@ -0,0 +1,26 @@
+## LedgerApp
+
+```
+class LedgerApp:
+
+    // sets the address to be used
+    set_address(account_index: int = 0, address_index: int = 0);
```

Comment:
Since we never want to allow people to use an `account_index` different from 0, maybe omit that parameter, and only allow one to provide  `address_index`?

CC: @ccorcoveanu

Reply:
Below, as well (we should consider account index as always zero, hardcoded; only address index allowed to vary for MultiversX).

Reply:
Done

Reply:
@andreibancioiu we don't?

Reply:
It was previously accepted as a param, but since it was only used within **mxpy** it was always set to `0`.